src/test/java/org/assertj/core/internal/files/Files_getFileContent_Test.java

```diff
@@ -0,0 +1,25 @@
+package org.assertj.core.internal.files;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.assertj.core.error.ShouldBeReadable.shouldBeReadable;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.assertj.core.api.AssertionInfo;
+import org.assertj.core.internal.FilesBaseTest;
+import org.junit.jupiter.api.Test;
+
+class Files_getFileContent_Test extends FilesBaseTest {
+  @Test
+  void should_fail_if_can_not_read() {
+    when(actual.canRead()).thenReturn(false);
+    AssertionInfo info = someInfo();
+
+    Throwable error = catchThrowable(() -> files.getFileContent(info, actual));
```

Comment:
use GIVEN  WHEN THEN as per  the [contributing guidelines](https://github.com/assertj/assertj-core/blob/main/CONTRIBUTING.md)